storage/src/iter.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright (C) 2023, Ava Labs, Inc. All rights reserved.
```

Comment:
This file is falsely included in the diff from a merge, I did not change anything here.